apps/graphql/src/apps/itinerary/types/outputs/ItineraryReturn.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import { GraphQLObjectType } from 'graphql';
+import { TRIP_TYPES } from '@kiwicom/margarita-config';
+
+import type { Itinerary } from '../../Itinerary';
+import ItineraryInterface, { commonFields } from './ItineraryInterface';
+
+export default new GraphQLObjectType({
+  name: 'ItineraryReturn',
+  description:
+    'Itinerary from A to B with no return, but with possible stopovers',
```

Comment:
The description isn't changed from the `OneWay` itinerary.

Reply:
Copy paste mistake. I will fix it ASAP